buildroot-external/kernel/device-support.config

```diff
@@ -37,12 +37,28 @@ CONFIG_USB_NET_SMSC95XX=m
 
 CONFIG_MEDIA_CEC_SUPPORT=y
 
+# Bluetooth HCI
 CONFIG_BT_INTEL=m
-CONFIG_BT_BCM=m
-CONFIG_BT_RTL=m
 CONFIG_BT_HCIBTUSB=m
+CONFIG_BT_HCIBTUSB_AUTOSUSPEND=y
```

Comment:
Are you sure that this doesn't lead to problems?

Reply:
Debian is doing that for years, if it doesn't sound right I'll just remove this :-)